plugins/modules/monitoring/datadog/datadog_downtime.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Datadog, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+---
+module: datadog_downtime
+short_description: Manages Datadog downtimes
+version_added: 2.0.0
+description:
+  - Manages downtimes within Datadog.
+  - Options as described on U(https://docs.datadoghq.com/api/v1/downtimes/s).
+author:
+  - Datadog (@Datadog)
+requirements: [datadog-api-client]
```

Comment:
Make the Python 3 dependency a bit more explicit:
```suggestion
requirements:
  - datadog-api-client
  - Python 3
```